[INFO] 2019-10-18 22:45:13,050 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-18 22:47:37,406 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-18 22:48:02,876 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 16:47:27,002 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 17:15:55,577 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 17:16:00,690 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:00] "GET / HTTP/1.1" 200 -
[INFO] 2019-10-19 17:16:00,801 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:00] "GET /favicon.ico HTTP/1.1" 404 -
[INFO] 2019-10-19 17:16:04,904 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:04] "GET / HTTP/1.1" 200 -
[ERROR] 2019-10-19 17:16:07,404 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [GET]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 17:16:07,404 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:07] "GET /login HTTP/1.1" 200 -
[INFO] 2019-10-19 17:16:07,450 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:07] "GET /favicon.ico HTTP/1.1" 404 -
[ERROR] 2019-10-19 17:16:11,045 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [GET]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 17:16:11,045 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:11] "GET /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 17:16:25,611 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([('', '')])
[INFO] 2019-10-19 17:16:25,611 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:16:25] "POST /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 17:18:55,803 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([('', '')])
[INFO] 2019-10-19 17:18:55,804 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:18:55] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 17:44:21,630 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 17:46:39,715 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:46:39] "POST / HTTP/1.1" 405 -
[INFO] 2019-10-19 17:46:58,910 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:46:58] "GET / HTTP/1.1" 200 -
[ERROR] 2019-10-19 17:47:46,857 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 17:47:46,859 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:47:46] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 17:53:26,649 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[ERROR] 2019-10-19 17:53:32,579 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 17:53:32,579 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:53:32] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 17:55:30,434 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[ERROR] 2019-10-19 17:55:35,467 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 17:55:35,468 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:55:35] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 17:56:05,799 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[ERROR] 2019-10-19 17:56:08,793 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[ERROR] 2019-10-19 17:56:08,794 app.py[line:1761] Exception on /login [POST]
Traceback (most recent call last):
  File "/Users/c/musharing_server/ser.py", line 84, in login
    name = request.form["name"]
  File "/usr/local/lib/python3.7/site-packages/werkzeug/datastructures.py", line 431, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.HTTPException.wrap.<locals>.newcls: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/c/musharing_server/ser.py", line 87, in login
    raise RequestError("not enough param")
  File "/Users/c/musharing_server/user_util.py", line 65, in __init__
    super(from_user)
TypeError: super() argument 1 must be type, not str

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/c/musharing_server/ser.py", line 105, in login
    print(e.with_traceback())
TypeError: with_traceback() takes exactly one argument (0 given)
[INFO] 2019-10-19 17:56:08,801 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:56:08] "POST /login HTTP/1.1" 500 -
[INFO] 2019-10-19 17:57:09,948 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[ERROR] 2019-10-19 17:57:12,928 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 17:57:12,929 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:57:12] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 17:58:11,951 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[ERROR] 2019-10-19 17:58:17,372 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[ERROR] 2019-10-19 17:58:17,373 app.py[line:1761] Exception on /login [POST]
Traceback (most recent call last):
  File "/Users/c/musharing_server/ser.py", line 84, in login
    name = request.form["name"]
  File "/usr/local/lib/python3.7/site-packages/werkzeug/datastructures.py", line 431, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.HTTPException.wrap.<locals>.newcls: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/c/musharing_server/ser.py", line 105, in login
    raise e
  File "/Users/c/musharing_server/ser.py", line 87, in login
    raise RequestError("not enough param")
  File "/Users/c/musharing_server/user_util.py", line 65, in __init__
    super(from_user)
TypeError: super() argument 1 must be type, not str
[INFO] 2019-10-19 17:58:17,381 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 17:58:17] "POST /login HTTP/1.1" 500 -
[INFO] 2019-10-19 18:00:09,275 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[ERROR] 2019-10-19 18:00:11,777 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[ERROR] 2019-10-19 18:00:11,778 app.py[line:1761] Exception on /login [POST]
Traceback (most recent call last):
  File "/Users/c/musharing_server/ser.py", line 84, in login
    name = request.form["name"]
  File "/usr/local/lib/python3.7/site-packages/werkzeug/datastructures.py", line 431, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.HTTPException.wrap.<locals>.newcls: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/c/musharing_server/ser.py", line 105, in login
    raise e
  File "/Users/c/musharing_server/ser.py", line 87, in login
    raise RequestError("not enough param")
user_util.RequestError: not enough param
[INFO] 2019-10-19 18:00:11,785 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 18:00:11] "POST /login HTTP/1.1" 500 -
[INFO] 2019-10-19 18:01:17,721 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 18:01:17,723 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 18:01:17,917 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 18:01:17,933 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 18:01:22,462 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 18:01:22,462 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 18:01:22] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 18:42:38,771 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 18:42:39,291 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 18:42:39,819 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 18:42:39,835 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 18:42:40,464 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 18:42:40,465 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 18:42:40] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 18:42:44,891 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 18:42:45,349 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 18:42:45,752 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 18:42:45,758 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 18:42:46,152 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 18:42:46,152 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 18:42:46] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 19:50:14,010 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 19:50:14,095 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 19:50:14,554 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 19:50:14,561 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 19:50:16,583 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 19:50:17,071 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 19:50:17,292 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 19:50:17,297 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 19:50:18,304 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 19:50:18,818 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 19:50:19,037 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 19:50:19,042 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 19:50:28,119 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 19:50:28,604 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 19:50:28,872 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 19:50:28,877 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 19:50:29,886 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 19:50:30,398 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 19:50:30,664 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 19:50:30,673 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 19:50:32,689 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 19:50:33,188 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 20:01:08,948 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 20:01:08,950 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:01:09,155 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:01:09,163 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:01:14,527 ser.py[line:60] <register>: success. uid = 3
[INFO] 2019-10-19 20:01:14,528 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:01:14] "POST /register HTTP/1.1" 200 -
[WARNING] 2019-10-19 20:01:53,215 ser.py[line:64] <register>: name_occupied. name = 6a2U6ay8
[INFO] 2019-10-19 20:01:53,215 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:01:53] "POST /register HTTP/1.1" 200 -
[INFO] 2019-10-19 20:02:52,994 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:02:52] "GET / HTTP/1.1" 200 -
[INFO] 2019-10-19 20:04:23,432 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:04:23] "POST /longin HTTP/1.1" 404 -
[ERROR] 2019-10-19 20:04:33,125 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([])
[INFO] 2019-10-19 20:04:33,125 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:04:33] "POST /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:05:24,318 ser.py[line:104] <login>: unexpected. request = <Request 'http://127.0.0.1:5000/login' [POST]>, request.form = ImmutableMultiDict([('name', '6a2U6ay8'), ('passwordddd', 'eff2133c6f622084ae62c109e9b8ea43')])
[INFO] 2019-10-19 20:05:24,320 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:05:24] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 20:05:28,935 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:05:28,936 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:05:28] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 20:05:50,849 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:05:50,849 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:05:50] "POST /login HTTP/1.1" 200 -
[WARNING] 2019-10-19 20:06:37,951 ser.py[line:96] <login>: wrong_password. name = 6a2U6ay8
[INFO] 2019-10-19 20:06:37,953 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:06:37] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 20:06:49,895 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:06:49,895 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:06:49] "POST /login HTTP/1.1" 200 -
[WARNING] 2019-10-19 20:06:55,459 ser.py[line:100] <login>: wrong_name. name = 6a2U6ay81
[INFO] 2019-10-19 20:06:55,459 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:06:55] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 20:07:00,046 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:07:00,048 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:07:00] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 20:14:23,759 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:23,893 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:24,302 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:24,307 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:25,320 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:25,831 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:26,088 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:26,093 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:28,119 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:28,649 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:28,856 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:28,860 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:30,880 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:31,386 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:31,583 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:31,587 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:43,694 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:44,178 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:44,380 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:44,384 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:46,408 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:46,922 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:47,123 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:47,127 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:55,213 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:55,687 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:55,909 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:55,914 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:14:57,935 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:14:58,439 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:14:58,671 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:14:58,676 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:17:40,602 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:17:40,636 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:17:40] "POST /login HTTP/1.1" 500 -
[INFO] 2019-10-19 20:20:09,864 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:20:10,431 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 20:21:28,262 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 20:21:28,264 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:21:28,468 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:21:28,475 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:21:34,972 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:21:34,974 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:21:34] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 20:23:16,441 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:16,978 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:17,346 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:17,353 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:18,367 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:18,882 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:19,135 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:19,142 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:22,181 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:22,669 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:22,895 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:22,900 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:23,912 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:24,426 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:24,640 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:24,645 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:25,656 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:26,172 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:26,401 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:26,405 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:27,419 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:27,935 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:28,268 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:28,273 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:35,347 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:35,841 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:36,102 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:36,107 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:40,154 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:40,645 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:40,897 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:40,909 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:44,961 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:45,441 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:45,659 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:45,664 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:47,686 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:48,197 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:48,455 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:48,460 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:49,472 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:49,982 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:50,186 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:50,191 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:56,257 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:56,744 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:56,988 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:56,993 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:23:58,009 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:23:58,521 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:23:58,731 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:23:58,735 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:03,790 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:04,284 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:04,538 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:04,543 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:06,564 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:07,069 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:07,295 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:07,299 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:11,343 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:11,852 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:12,086 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:12,091 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:13,106 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:13,616 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:13,839 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:13,844 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:21,961 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:22,398 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:22,657 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:22,664 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:24,695 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:25,192 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:25,421 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:25,426 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:29,467 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:29,968 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:30,248 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:30,254 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:32,279 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:32,785 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:32,997 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:33,001 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:35,026 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:35,547 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:35,821 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:35,825 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:38,857 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:39,358 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:39,585 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:39,590 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:43,633 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:44,135 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:44,360 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:44,365 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:46,387 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:46,916 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:47,174 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:47,179 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:49,197 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:49,700 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:49,935 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:49,940 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:50,955 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:51,465 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:51,674 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:51,678 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:56,738 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:57,220 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:57,427 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:57,432 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:24:58,450 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:24:58,959 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:24:59,169 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:24:59,174 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:25:02,207 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:25:02,710 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:25:02,955 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:25:02,960 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:25:04,986 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:25:05,490 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:25:05,698 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:25:05,703 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:25:17,820 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:25:18,289 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:25:18,520 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:25:18,525 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:25:31,658 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:25:32,107 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:25:32,387 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:25:32,392 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:25:33,404 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/response_util.py', reloading
[INFO] 2019-10-19 20:25:33,921 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:25:34,154 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:25:34,158 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:28:29,744 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:28:29,746 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:28:29] "POST /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:29:36,334 ser.py[line:338] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('name', '6a2U6ay8'), ('uid', '3')])
[INFO] 2019-10-19 20:29:36,335 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:29:36] "POST /members HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:29:52,945 ser.py[line:327] <members>: from_not_login. from_uid = 3
[INFO] 2019-10-19 20:29:52,947 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:29:52] "POST /members HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:30:05,692 ser.py[line:327] <members>: from_not_login. from_uid = 3
[INFO] 2019-10-19 20:30:05,692 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:30:05] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 20:30:51,700 ser.py[line:92] <login>: success. uid = 3
[INFO] 2019-10-19 20:30:51,702 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:30:51] "POST /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:31:10,709 ser.py[line:327] <members>: from_not_login. from_uid = 3
[INFO] 2019-10-19 20:31:10,710 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:31:10] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 20:34:19,043 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:19,376 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:19,789 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:19,794 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:34:20,805 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:21,309 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:21,549 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:21,555 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:34:22,563 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:23,097 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:23,451 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:23,456 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:34:25,476 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:25,965 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:26,172 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:26,177 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:34:31,221 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:31,708 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:31,969 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:31,975 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:34:33,997 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:34,493 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:34,699 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:34,703 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:34:38,741 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:34:39,246 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:34:39,477 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:34:39,482 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:35:09,446 ser.py[line:93] <login>: success. uid = 3
[INFO] 2019-10-19 20:35:09,446 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:35:09] "POST /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:35:21,283 ser.py[line:332] <members>: from_not_in_room. from_uid = 3
[INFO] 2019-10-19 20:35:21,283 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:35:21] "POST /members HTTP/1.1" 200 -
[CRITICAL] 2019-10-19 20:35:50,633 ser.py[line:324] <members>: from_not_exist. from_uid = 30
[INFO] 2019-10-19 20:35:50,634 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:35:50] "POST /members HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:35:53,812 ser.py[line:332] <members>: from_not_in_room. from_uid = 3
[INFO] 2019-10-19 20:35:53,812 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:35:53] "POST /members HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:36:28,044 ser.py[line:332] <attend>: from_not_in_room. from_uid = 3
[INFO] 2019-10-19 20:36:28,044 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:36:28] "POST /attend HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:36:32,402 ser.py[line:332] <attend>: from_not_in_room. from_uid = 3
[INFO] 2019-10-19 20:36:32,402 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:36:32] "POST /attend HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:36:39,838 ser.py[line:332] <attend>: from_not_in_room. from_uid = 3
[INFO] 2019-10-19 20:36:39,838 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:36:39] "POST /attend HTTP/1.1" 200 -
[ERROR] 2019-10-19 20:36:40,832 ser.py[line:332] <attend>: from_not_in_room. from_uid = 3
[INFO] 2019-10-19 20:36:40,832 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:36:40] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 20:40:00,347 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:00,437 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:00,765 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:00,769 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:40:28,017 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:28,376 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:28,602 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:28,606 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:40:32,641 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:33,154 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:33,441 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:33,445 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:40:41,514 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:42,009 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:42,244 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:42,250 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:40:47,295 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:47,786 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:48,035 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:48,040 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:40:49,048 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:49,562 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:49,771 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:49,776 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:40:52,798 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:40:53,301 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:40:53,581 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:40:53,587 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:41:05,697 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:41:06,135 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:41:06,393 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:41:06,399 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:41:46,753 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:41:47,087 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:41:47,312 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:41:47,316 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:42:19,596 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:42:19,949 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:42:20,192 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:42:20,196 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:42:40,368 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:42:40,794 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:42:41,025 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:42:41,030 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:04,221 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:04,666 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:04,944 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:04,950 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:05,964 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:06,461 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:06,706 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:06,711 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:10,750 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:11,253 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:11,469 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:11,474 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:12,483 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:12,995 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:13,217 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:13,221 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:18,262 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:18,769 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:18,981 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:18,986 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:28,065 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:28,546 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:28,772 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:28,778 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:43:58,028 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:43:58,392 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:43:58,616 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:43:58,621 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:44:30,914 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:44:31,274 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:44:31,486 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:44:31,490 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:44:35,524 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:44:36,028 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:44:36,244 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:44:36,248 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:44:38,265 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:44:38,771 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:44:38,989 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:44:38,993 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:44:44,037 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:44:44,536 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:44:44,744 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:44:44,749 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:44:46,769 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:44:47,275 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:44:47,488 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:44:47,492 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:44:58,580 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:44:59,044 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:44:59,299 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:44:59,304 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:45:03,338 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:45:03,830 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:45:04,075 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:45:04,080 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:45:08,111 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:45:08,617 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:45:08,840 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:45:08,845 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:45:09,852 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:45:10,369 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:45:10,579 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:45:10,584 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:45:20,683 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:45:21,144 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:45:21,371 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:45:21,376 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 20:45:50,628 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:45:51,003 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:45:51,238 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:45:51,243 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 20:46:05,981 ser.py[line:368] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 20:46:05,982 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:46:05] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 20:52:25,851 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 20:52:26,183 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 20:52:54,381 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 20:52:54,382 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 20:52:54,574 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 20:52:54,580 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 20:52:57,937 ser.py[line:368] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 20:52:57,950 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 20:52:57] "POST /attend HTTP/1.1" 500 -
[INFO] 2019-10-19 21:21:40,518 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:21:41,035 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:21:41,360 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:21:41,367 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:21:42,378 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:21:42,897 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:21:43,170 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:21:43,176 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:21:48,148 ser.py[line:370] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:21:48,160 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:21:48] "POST /attend HTTP/1.1" 500 -
[INFO] 2019-10-19 21:24:14,646 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:24:15,170 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:24:15,414 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:24:15,419 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:24:16,429 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:24:16,948 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:24:17,165 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:24:17,170 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:24:23,224 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:24:23,717 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:24:23,915 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:24:23,920 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:24:41,093 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:24:41,533 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:24:41,788 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:24:41,792 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:24:43,811 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:24:44,322 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:24:44,526 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:24:44,530 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:24:53,609 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:24:54,090 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:24:54,315 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:24:54,320 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:25:02,397 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:25:02,875 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:25:03,099 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:25:03,104 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:25:23,303 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:25:23,719 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:25:23,951 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:25:23,965 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:25:24,982 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:25:25,489 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:25:25,773 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:25:25,777 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:25:33,844 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:25:34,344 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:25:34,545 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:25:34,550 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:25:38,582 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:25:39,087 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:25:39,309 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:25:39,314 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:25:43,676 ser.py[line:370] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:25:43,686 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:25:43] "POST /attend HTTP/1.1" 500 -
[INFO] 2019-10-19 21:26:17,671 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:26:18,035 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:26:18,359 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:26:18,367 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:26:22,403 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:26:22,886 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:26:23,094 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:26:23,099 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:26:23,739 ser.py[line:368] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:26:23,740 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:26:23] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:27:12,288 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = 345
[INFO] 2019-10-19 21:27:12,289 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:27:12] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:27:26,906 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = foo
[INFO] 2019-10-19 21:27:26,906 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:27:26] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:28:58,007 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = Zm9v
[INFO] 2019-10-19 21:28:58,011 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:28:58] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:28:59,117 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = Zm9v
[INFO] 2019-10-19 21:28:59,117 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:28:59] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:29:04,290 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = Zm9v\n
[INFO] 2019-10-19 21:29:04,291 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:29:04] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:30:16,792 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = Rm9v
[INFO] 2019-10-19 21:30:16,793 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:30:16] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:30:22,344 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = Rm9v\n
[INFO] 2019-10-19 21:30:22,344 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:30:22] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:30:26,706 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = Rm9v\n
[INFO] 2019-10-19 21:30:26,707 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:30:26] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:30:50,442 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = QmFy\n
[INFO] 2019-10-19 21:30:50,443 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:30:50] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:30:59,271 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = QmFy
[INFO] 2019-10-19 21:30:59,271 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:30:59] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 21:34:15,648 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:34:15,983 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:34:16,340 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:34:16,347 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:34:21,401 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:34:21,880 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:34:22,078 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:34:22,082 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:34:26,125 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:34:26,623 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:34:26,849 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:34:26,853 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:34:30,704 ser.py[line:368] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3'), ('target_name', 'QmFy')])
[INFO] 2019-10-19 21:34:30,705 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:34:30] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 21:35:24,372 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:35:24,722 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:35:25,092 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:35:25,098 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:35:41,249 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:35:41,688 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:35:41,896 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:35:41,902 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:35:43,918 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:35:44,430 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:35:44,650 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:35:44,656 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:35:49,713 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:35:50,209 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:35:50,408 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:35:50,413 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:36:18,662 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:36:19,092 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:36:19,314 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:36:19,321 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:36:20,875 ser.py[line:368] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3'), ('target_name', 'QmFy')])
[INFO] 2019-10-19 21:36:20,885 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:36:20] "POST /attend HTTP/1.1" 500 -
[INFO] 2019-10-19 21:37:47,148 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:37:47,346 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:37:47,697 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:37:47,703 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:38:09,916 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:38:10,316 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:38:10,534 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:38:10,538 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:38:13,567 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/authenticate_util.py', reloading
[INFO] 2019-10-19 21:38:14,073 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:38:14,304 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:38:14,309 _internal.py[line:88]  * Debugger PIN: 751-962-697
[WARNING] 2019-10-19 21:38:20,945 ser.py[line:117] <attend>: target_not_exist. from_uid = 3, target_name = QmFy
[INFO] 2019-10-19 21:38:20,945 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:38:20] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 21:38:33,300 ser.py[line:148] <attend>: success. 3 :+ 3 -> 2fcbd032-6929-357c-8c46-f584cf8b0d1f
[INFO] 2019-10-19 21:38:33,300 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:38:33] "POST /attend HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:42:06,230 ser.py[line:101] <login>: wrong_name. name = a
[INFO] 2019-10-19 21:42:06,231 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:42:06] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 21:42:18,406 ser.py[line:60] <register>: success. uid = 4
[INFO] 2019-10-19 21:42:18,407 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:42:18] "POST /register HTTP/1.1" 200 -
[WARNING] 2019-10-19 21:42:33,437 ser.py[line:121] <attend>: target_not_login. from_uid = 3, target_name = a
[INFO] 2019-10-19 21:42:33,437 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:42:33] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 21:42:38,740 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-19 21:42:38,740 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:42:38] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 21:42:42,987 ser.py[line:148] <attend>: success. 3 :+ 4 -> 2fcbd032-6929-357c-8c46-f584cf8b0d1f
[INFO] 2019-10-19 21:42:42,987 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:42:42] "POST /attend HTTP/1.1" 200 -
[ERROR] 2019-10-19 21:43:49,087 ser.py[line:320] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:43:49,088 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:43:49] "POST /members HTTP/1.1" 200 -
[ERROR] 2019-10-19 21:44:03,669 ser.py[line:320] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:44:03,669 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:44:03] "POST /members HTTP/1.1" 200 -
[ERROR] 2019-10-19 21:47:59,167 ser.py[line:320] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:47:59,168 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:47:59] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 21:48:19,063 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:48:19,416 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:48:19,741 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:48:19,746 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:50:00,726 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:50:00,894 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:50:01,155 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:50:01,160 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:50:04,190 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:50:04,696 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:50:04,951 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:50:04,956 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:50:06,971 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:50:07,486 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:50:07,712 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:50:07,718 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:50:16,792 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:50:17,274 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:50:17,499 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:50:17,506 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:50:23,563 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:50:24,059 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:50:24,271 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:50:24,278 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:50:35,474 ser.py[line:322] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:50:35,474 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:50:35] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 21:51:06,679 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:51:07,022 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:51:07,246 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:51:07,252 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:51:11,290 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:51:11,790 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:51:12,049 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:51:12,053 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:51:14,071 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:51:14,585 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:51:14,846 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:51:14,850 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:51:17,877 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:51:18,388 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:51:18,667 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:51:18,672 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:51:37,919 ser.py[line:322] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:51:37,920 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:51:37] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 21:52:04,080 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:52:04,394 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:52:04,613 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:52:04,619 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:52:09,980 ser.py[line:322] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:52:09,980 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:52:09] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 21:52:32,896 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:52:33,278 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:52:33,587 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:52:33,593 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:52:36,620 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:52:37,121 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:52:37,438 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:52:37,443 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 21:52:51,153 ser.py[line:319] <members>: unexpected. request = <Request 'http://127.0.0.1:5000/members' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 21:52:51,153 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 21:52:51] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 21:53:50,110 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:53:50,402 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:53:50,716 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:53:50,720 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:53:59,806 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:54:00,293 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:54:00,540 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:54:00,546 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:54:03,576 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:54:04,083 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 21:54:04,289 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 21:54:04,294 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 21:54:20,437 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 21:54:20,906 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 22:00:06,663 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:00:06,664 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:00:06,869 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:00:06,876 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 22:00:10,873 ser.py[line:309] <members>: from_not_login. from_uid = 3
[INFO] 2019-10-19 22:00:10,874 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:00:10] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 22:00:17,840 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-19 22:00:17,841 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:00:17] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:00:37,158 ser.py[line:60] <register>: success. uid = 5
[INFO] 2019-10-19 22:00:37,159 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:00:37] "POST /register HTTP/1.1" 200 -
[INFO] 2019-10-19 22:00:44,675 ser.py[line:93] <login>: success. uid = 5
[INFO] 2019-10-19 22:00:44,676 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:00:44] "POST /login HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:01:28,812 ser.py[line:369] <attend>: unexpected. request = <Request 'http://127.0.0.1:5000/attend' [POST]>, request.form = ImmutableMultiDict([('from_uid', '3')])
[INFO] 2019-10-19 22:01:28,812 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:01:28] "POST /attend HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:01:45,541 ser.py[line:362] <attend>: from_not_login. from_uid = 3
[INFO] 2019-10-19 22:01:45,541 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:01:45] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 22:01:55,827 ser.py[line:93] <login>: success. uid = 5
[INFO] 2019-10-19 22:01:55,828 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:01:55] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:02:07,767 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-19 22:02:07,768 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:02:07] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:02:19,940 ser.py[line:148] <attend>: success. 4 :+ 5 -> fb8b90dc-ebc9-3fc8-b703-6b486bcd937e
[INFO] 2019-10-19 22:02:19,941 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:02:19] "POST /attend HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:02:24,614 ser.py[line:309] <members>: from_not_login. from_uid = 3
[INFO] 2019-10-19 22:02:24,615 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:02:24] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 22:02:28,943 ser.py[line:163] <members>: success. from_uid = 4
[INFO] 2019-10-19 22:02:28,943 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:02:28] "POST /members HTTP/1.1" 200 -
[WARNING] 2019-10-19 22:03:19,719 ser.py[line:125] <attend>: target_in_room. from_uid = 4, target_name = b
[INFO] 2019-10-19 22:03:19,719 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:03:19] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 22:05:31,065 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-19 22:05:31,065 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:05:31] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-19 22:05:42,175 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-19 22:05:42,175 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:05:42] "POST /category HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:10:08,154 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4')])
[INFO] 2019-10-19 22:10:08,155 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:10:08] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:10:31,596 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist', '234')])
[INFO] 2019-10-19 22:10:31,598 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:10:31] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:10:37,410 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist', '234')])
[INFO] 2019-10-19 22:10:37,410 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:10:37] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:10:38,129 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist', '234')])
[INFO] 2019-10-19 22:10:38,130 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:10:38] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:11:12,893 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist', '234')])
[INFO] 2019-10-19 22:11:12,896 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:11:12] "POST /playlist HTTP/1.1" 200 -
[WARNING] 2019-10-19 22:11:51,756 ser.py[line:249] <playlist> playlist_not_exist. playlist_id = 234, from_uid = 4
[INFO] 2019-10-19 22:11:51,756 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:11:51] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:12:09,788 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:12:09,788 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:12:09] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:12:33,131 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:12:33,132 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:12:33] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-19 22:13:09,200 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:13:09,466 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:13:09,972 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:13:09,978 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:13:10,988 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:13:11,487 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:13:11,696 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:13:11,701 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:13:15,729 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:13:16,236 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:13:16,464 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:13:16,469 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:13:24,539 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:13:25,030 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:13:25,260 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:13:25,265 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:13:31,322 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:13:31,804 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:13:32,024 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:13:32,029 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:13:36,066 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:13:36,558 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:13:36,778 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:13:36,783 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 22:13:40,516 ser.py[line:321] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:13:40,517 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:13:40] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:16:48,320 ser.py[line:321] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:16:48,321 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:16:48] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-19 22:17:00,741 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:17:01,100 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:17:01,591 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:17:01,597 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:17:35,927 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:17:36,278 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:17:36,482 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:17:36,486 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:17:42,534 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:17:43,060 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:17:43,289 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:17:43,293 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 22:17:47,650 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:17:47,662 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:17:47] "POST /playlist HTTP/1.1" 500 -
[INFO] 2019-10-19 22:23:17,107 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:23:17,497 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 22:24:21,433 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:24:21,434 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:24:21,635 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:24:21,643 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 22:24:23,246 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:24:23,257 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:24:23] "POST /playlist HTTP/1.1" 500 -
[INFO] 2019-10-19 22:25:20,657 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-19 22:25:20,658 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:25:20] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-19 22:26:29,774 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:26:29,855 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 22:26:47,959 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:26:47,960 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:26:48,156 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:26:48,160 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:26:50,421 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-19 22:26:50,422 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:26:50] "POST /category HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:26:52,650 ser.py[line:320] <playlist>: unexpected. request = <Request 'http://127.0.0.1:5000/playlist' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('playlist_id', '1')])
[INFO] 2019-10-19 22:26:52,662 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:26:52] "POST /playlist HTTP/1.1" 500 -
[INFO] 2019-10-19 22:32:08,257 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:32:08,259 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:32:08,476 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:32:08,484 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:32:11,935 ser.py[line:246] <playlist> success. playlist_id = 1, from_uid = 4
[INFO] 2019-10-19 22:32:11,935 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:32:11] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-19 22:32:43,810 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:32:44,194 _internal.py[line:88]  * Restarting with stat
[INFO] 2019-10-19 22:32:49,514 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:32:49,515 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:32:49,702 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:32:49,707 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:32:52,907 ser.py[line:246] <playlist> success. playlist_id = 1, from_uid = 4
[INFO] 2019-10-19 22:32:52,908 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:32:52] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-19 22:33:05,507 ser.py[line:246] <playlist> success. playlist_id = 1, from_uid = 4
[INFO] 2019-10-19 22:33:05,508 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:33:05] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-19 22:33:14,331 ser.py[line:246] <playlist> success. playlist_id = 0, from_uid = 4
[INFO] 2019-10-19 22:33:14,331 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:33:14] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-19 22:33:44,370 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:33:44,371 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:33:44,585 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:33:44,590 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:33:49,127 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-19 22:33:49,128 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:33:49] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:33:52,982 ser.py[line:93] <login>: success. uid = 5
[INFO] 2019-10-19 22:33:52,983 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:33:52] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:33:57,825 ser.py[line:148] <attend>: success. 4 :+ 5 -> a731bc2a-421c-353f-a02c-301027f6edcd
[INFO] 2019-10-19 22:33:57,825 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:33:57] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 22:34:01,108 ser.py[line:163] <members>: success. from_uid = 4
[INFO] 2019-10-19 22:34:01,109 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:34:01] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 22:34:05,786 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-19 22:34:05,786 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:34:05] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-19 22:34:08,203 ser.py[line:246] <playlist> success. playlist_id = 0, from_uid = 4
[INFO] 2019-10-19 22:34:08,204 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:34:08] "POST /playlist HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:35:23,572 ser.py[line:320] <send>: unexpected. request = <Request 'http://127.0.0.1:5000/send' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4')])
[INFO] 2019-10-19 22:35:23,584 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:35:23] "POST /send HTTP/1.1" 500 -
[INFO] 2019-10-19 22:35:34,648 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-19 22:35:34,785 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:35:35,161 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:35:35,165 _internal.py[line:88]  * Debugger PIN: 751-962-697
[ERROR] 2019-10-19 22:35:41,206 ser.py[line:320] <send>: unexpected. request = <Request 'http://127.0.0.1:5000/send' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4')])
[INFO] 2019-10-19 22:35:41,207 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:35:41] "POST /send HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:35:51,702 ser.py[line:320] <send>: unexpected. request = <Request 'http://127.0.0.1:5000/send' [POST]>, request.form = ImmutableMultiDict([('from_uid', '4'), ('msg', '123')])
[INFO] 2019-10-19 22:35:51,703 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:35:51] "POST /send HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:03,598 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-19 22:36:03,599 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-19 22:36:03,802 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-19 22:36:03,806 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-19 22:36:09,917 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-19 22:36:09,918 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:09] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:12,449 ser.py[line:93] <login>: success. uid = 5
[INFO] 2019-10-19 22:36:12,450 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:12] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:14,588 ser.py[line:148] <attend>: success. 4 :+ 5 -> d7b4d38d-19c0-32d7-b89d-477489a136de
[INFO] 2019-10-19 22:36:14,588 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:14] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:16,838 ser.py[line:163] <members>: success. from_uid = 4
[INFO] 2019-10-19 22:36:16,838 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:16] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:24,337 ser.py[line:178] <send>: success. from_uid = 4
[INFO] 2019-10-19 22:36:24,337 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:24] "POST /send HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:29,669 ser.py[line:178] <send>: success. from_uid = 4
[INFO] 2019-10-19 22:36:29,669 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:29] "POST /send HTTP/1.1" 200 -
[INFO] 2019-10-19 22:36:55,079 ser.py[line:178] <send>: success. from_uid = 4
[INFO] 2019-10-19 22:36:55,080 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:36:55] "POST /send HTTP/1.1" 200 -
[INFO] 2019-10-19 22:37:28,837 ser.py[line:193] <receive>: success. size = 3; from_uid = 4
[INFO] 2019-10-19 22:37:28,837 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:37:28] "POST /receive HTTP/1.1" 200 -
[INFO] 2019-10-19 22:38:15,206 ser.py[line:211] <leave> success. d7b4d38d-19c0-32d7-b89d-477489a136de -= 4
[INFO] 2019-10-19 22:38:15,208 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:38:15] "POST /leave HTTP/1.1" 200 -
[ERROR] 2019-10-19 22:38:22,398 ser.py[line:313] <members>: from_not_in_room. from_uid = 4
[INFO] 2019-10-19 22:38:22,398 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:38:22] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 22:38:26,190 ser.py[line:163] <members>: success. from_uid = 5
[INFO] 2019-10-19 22:38:26,190 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:38:26] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-19 22:39:09,272 ser.py[line:232] <logout>: success. from_uid = 4
[INFO] 2019-10-19 22:39:09,272 _internal.py[line:88] 127.0.0.1 - - [19/Oct/2019 22:39:09] "POST /logout HTTP/1.1" 200 -
[INFO] 2019-10-20 01:43:37,560 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-20 01:43:37,561 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:43:37,833 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:43:37,845 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:43:53,821 ser.py[line:93] <login>: success. uid = 5
[INFO] 2019-10-20 01:43:53,821 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:43:53] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-20 01:44:05,856 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-20 01:44:05,856 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:44:05] "POST /login HTTP/1.1" 200 -
[WARNING] 2019-10-20 01:44:10,506 ser.py[line:125] <attend>: target_in_room. from_uid = 4, target_name = b
[INFO] 2019-10-20 01:44:10,506 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:44:10] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-20 01:44:37,496 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-20 01:44:37,496 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:44:37,704 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:44:37,709 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:44:50,388 ser.py[line:93] <login>: success. uid = 4
[INFO] 2019-10-20 01:44:50,389 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:44:50] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-20 01:44:53,557 ser.py[line:93] <login>: success. uid = 5
[INFO] 2019-10-20 01:44:53,557 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:44:53] "POST /login HTTP/1.1" 200 -
[INFO] 2019-10-20 01:44:57,240 ser.py[line:148] <attend>: success. 4 :+ 5 -> 07f47ac1-1a0d-3476-aee3-98ea44c813a9
[INFO] 2019-10-20 01:44:57,240 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:44:57] "POST /attend HTTP/1.1" 200 -
[INFO] 2019-10-20 01:45:02,300 ser.py[line:163] <members>: success. from_uid = 5
[INFO] 2019-10-20 01:45:02,300 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:45:02] "POST /members HTTP/1.1" 200 -
[INFO] 2019-10-20 01:45:14,373 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-20 01:45:14,374 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:45:14] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-20 01:46:24,723 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:46:24,933 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:46:25,370 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:46:25,376 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:46:28,400 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:46:28,910 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:46:29,148 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:46:29,154 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:46:31,694 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-20 01:46:31,694 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:46:31] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-20 01:46:41,174 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-20 01:46:41,175 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:46:41] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-20 01:46:42,483 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-20 01:46:42,483 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:46:42] "POST /category HTTP/1.1" 200 -
[INFO] 2019-10-20 01:47:34,768 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:47:35,047 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:47:35,544 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:47:35,553 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:18,938 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:19,288 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:19,526 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:19,539 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:20,554 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:21,075 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:21,288 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:21,293 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:22,303 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:22,817 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:23,035 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:23,039 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:27,072 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:27,580 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:27,797 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:27,801 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:29,821 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:30,339 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:30,549 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:30,554 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:32,572 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:33,088 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:33,396 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:33,405 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:53,594 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:54,044 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:54,415 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:54,421 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:48:57,448 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:48:57,957 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:48:58,169 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:48:58,173 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:49:12,322 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:49:12,785 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:49:13,024 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:49:13,030 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:49:16,055 _internal.py[line:88]  * Detected change in '/Users/c/musharing_server/ser.py', reloading
[INFO] 2019-10-20 01:49:16,571 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-20 01:49:16,841 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-20 01:49:16,846 _internal.py[line:88]  * Debugger PIN: 751-962-697
[INFO] 2019-10-20 01:49:23,026 ser.py[line:263] <category> success. from_uid = 4
[INFO] 2019-10-20 01:49:23,026 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:49:23] "POST /category HTTP/1.1" 200 -
[WARNING] 2019-10-20 01:50:04,848 ser.py[line:249] <playlist> playlist_not_exist. playlist_id = 33, from_uid = 4
[INFO] 2019-10-20 01:50:04,848 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:50:04] "POST /playlist HTTP/1.1" 200 -
[WARNING] 2019-10-20 01:50:11,267 ser.py[line:249] <playlist> playlist_not_exist. playlist_id = 0, from_uid = 4
[INFO] 2019-10-20 01:50:11,267 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:50:11] "POST /playlist HTTP/1.1" 200 -
[WARNING] 2019-10-20 01:50:15,159 ser.py[line:249] <playlist> playlist_not_exist. playlist_id = 0, from_uid = 4
[INFO] 2019-10-20 01:50:15,161 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:50:15] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-20 01:50:18,104 ser.py[line:246] <playlist> success. playlist_id = 1, from_uid = 4
[INFO] 2019-10-20 01:50:18,104 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:50:18] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-20 01:50:30,738 ser.py[line:246] <playlist> success. playlist_id = 2, from_uid = 4
[INFO] 2019-10-20 01:50:30,738 _internal.py[line:88] 127.0.0.1 - - [20/Oct/2019 01:50:30] "POST /playlist HTTP/1.1" 200 -
[INFO] 2019-10-21 12:44:26,433 _internal.py[line:88]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[INFO] 2019-10-21 12:44:26,435 _internal.py[line:88]  * Restarting with stat
[WARNING] 2019-10-21 12:44:26,632 _internal.py[line:88]  * Debugger is active!
[INFO] 2019-10-21 12:44:26,650 _internal.py[line:88]  * Debugger PIN: 751-962-697
